dojo/tasks.py

```diff
@@ -382,3 +382,9 @@ def send_hipchat_notification_task(*args, **kwargs):
 def send_alert_notification_task(*args, **kwargs):
     logger.debug("send_alert_notification")
     send_alert_notification(*args, **kwargs)
+
+
+@app.task(name='dojo.tasks.async_sla_compute_and_notify')
+def async_sla_compute_and_notify_task(*args, **kwargs):
+    logger.debug("Computing SLAs and notifying as needed")
+    sla_compute_and_notify(*args, **kwargs)
```

Comment:
nit: maybe you should catch and handle all the exception which might be eventually thrown by this function in order to avoid the cron job from crashing

Reply:
Added try/catch, thanks ;-)